src/vsphere_cpi/spec/integration/cpi_extension_spec.rb

```diff
@@ -41,6 +44,40 @@
     end
   end
 
+  context 'when it replicates a stemcell on vsphere' do
+    let(:second_cluster_cpi) do
+      options = cpi_options(
+        datacenters: [{
+          clusters: [@cluster_name],
+          datastore_pattern: @datastore_pattern,
+          persistent_datastore_pattern: @datastore_pattern
+        }],
+      )
+      VSphereCloud::Cloud.new(options)
+    end
+
+    it 'adds replicated stemcell to extension' do
```

Comment:
'add replicated stemcell and vm to extension' 
since you are testing that vm also is added to extension in this test, you can essentially delete this test which is now redundant  'adds vm to the default cpi extension''